### DISCUSSION TITLE
Packaging error

### DISCUSSION THREAD

**Discussion by centurydong** [electron-vite/electron-vite-vue#165] [Q&A]
When running the "NPM run build" command, I encountered the following error:
### cannot move downloaded into final location (another process downloaded faster?)  
  path=C:\Users\xxx\AppData\Local\electron-builder\Cache\nsis\nsis-resources-3.4.1
  tempFile=C:\Users\xxx\AppData\Local\electron-builder\Cache\nsis\587137555
  error=rename C:\Users\xxx\AppData\Local\electron-builder\Cache\nsis\587137555 C:\Users\xxx\AppData\Local\electron-builder\Cache\nsis\nsis-resources-3.4.1: **Access is denied.**

### Can you give me some help? Thank you.

**Comment by caoxiemeihao** [comment]
This looks like the resource handle is occupied. You can try to "run CMD as administrator" and the execute `npm run build`.  
*这看起来像资源句柄被占用，你可以尝试 “以为管理员身份运行 CMD”，然后执行 `npm run build`*
